buildpacks/hello-processes/bin/detect

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+# Nothing to see here. :)
+# This buildpack applies unconditionally.
```

Comment:
do you think an explicit `exit 0` would be helpful here?

Reply:
It did cross my mind. Wasn't sure what was best but since you mention it I'll go ahead and add it.